packages/react-components/src/components/StatusBadge/StatusBadge.module.scss

```diff
@@ -0,0 +1,79 @@
+$base-class: 'status-badge';
+
+@mixin size($size) {
+  border-radius: $size;
+  width: $size;
+  height: $size;
+}
+
+.#{$base-class} {
+  display: flex;
+  border-style: solid;
+  border-color: var(--surface-primary-default);
+
+  &--3XS {
+    @include size(4px);
+
+    border-width: 1px;
```

Comment:
I'd suggest including border-width into the size mixin and make it as a second parameter